Apps/W1/ShopifyConnector/app/Order handeling/Codeunits/ShpfyImportOrder.Codeunit.al

```diff
@@ -0,0 +1,646 @@
+/// <summary>
+/// Codeunit Shpfy Import Order (ID 30161).
+/// </summary>
+codeunit 30161 "Shpfy Import Order"
+{
+    Access = Internal;
+    TableNo = "Shpfy Orders To Import";
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"